frontend/src/views/FinishedView.vue

```diff
@@ -0,0 +1,175 @@
+<!-- View for a finished sequence -->
+<template>
+  <div class="flex flex-col">
+    <img src="../assets/Logo.png" class="w-1/2 mx-auto" />
+    <div class="text-center text-2xl">
+      Du hast es geschafft<br />
+      und die Lernsequenz "{{ sequence.name }}" vollständig bearbeitet!
+    </div>
+  </div>
+  <div class="firework overflow-hidden"></div>
+  <div class="firework overflow-hidden"></div>
+  <div class="firework overflow-hidden"></div>
+</template>
+
+<script setup lang="ts">
+import { SequenceRestInterface } from '../restInterfaces/SequenceRestInterface';
+import { ref } from 'vue';
+
+const prop = defineProps({
+  /**
+   * Sequence code to get meta data
+   */
+  code: {
+    type: String,
+    required: true,
+  },
```

Comment:
Do we want trailing commas?

Reply:
As far as I know, prettier automatically adds trailing commas where possible, it also makes adding new entries to an object easier